qark/issue.py

```diff
@@ -47,10 +50,22 @@ class Severity(Enum):
     VULNERABILITY = 3
 
 
-class VulnerabilityEncoder(JSONEncoder):
-    def default(self, vulnerability):
-        return dict((('severity', vulnerability.severity.name), ('name', vulnerability.issue_name)))
+class IssueEncoder(JSONEncoder):
+    def default(self, issue):
+        if isinstance(issue, Issue):
+            return dict((('category', issue.category),
```

Comment:
What do you think about `json.dumps(issue.__dict__)`? It would be so that we never have to touch this section as long as everything in the dict can be serializable (it is right now).